raw-ipa-lib/src/helpers/event.rs

```diff
@@ -92,3 +109,88 @@ impl DerefMut for Helper {
         &mut self.public
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{Helper, Role};
+    use crate::user::User;
+    use crate::user::EventReport;
+        use crate::threshold::{
+        DecryptionKey as ThresholdDecryptionKey, EncryptionKey as ThresholdEncryptionKey,
+    };
+    pub use rust_elgamal::{Ciphertext, EncryptionKey, RistrettoPoint, Scalar, GENERATOR_POINT};
+    use rand::thread_rng;
+
+    #[test]
+    fn test_the_basics() {
+        let h_se = Helper::new(Role::Source);
+        let h_te = Helper::new(Role::Trigger);
+
+        let tek = ThresholdEncryptionKey::new(&[h_se.public.matchkey_encrypt, h_te.public.matchkey_encrypt]);
+
+        const PROVIDER_1: &str = "social.example";
+        const PROVIDER_2: &str = "news.example";
+        const PROVIDER_3: &str = "email.example";
+        const PROVIDER_4: &str = "game.example";
+
+        const MATCHING_MATCHKEY: &str = "12345678";
+
+        let mut u123 = User::new(123, tek);
```

Comment:
Given that the user ID doesn't matter, maybe just make it
```suggestion
        let mut u1 = User::new(1, tek);
        let mut u2 = User::new(2, tek);
```

That might silence at least *some* of clippy's complaints here.

Reply:
ok - will do.